internal/host/conditions.go

```diff
@@ -32,6 +33,11 @@ func (v *validator) isInstallationDiskSpeedCheckSuccessful(c *validationContext)
 	if c.infraEnv != nil {
 		return false
 	}
+
+	if hostutil.SaveDiskPartitionsIsSet(c.host.InstallerArgs) {
```

Comment:
Is there any way we could skip this in the agent and return a successful result?
Then we wouldn't have to check this everywhere the speed check matters, right?

I assume we can't because we don't have the installer args in the speed check command?

Reply:
Another way is to add a fake result into the DB, then the rest of the code will see that it was already checked and will skip the check.

Reply:
@carbonin we can't as you said. 
@avishayt pushing fake values will require removing them in case install args will be changed. Not sure we want to enter this logic

Reply:
we can provide installer args to agent and lets it decide what to do with installer command and which result to return

Reply:
I'm OK with the code as it is, just trying to address @carbonin 's concern

Reply:
Not a big concern. If it's difficult I wouldn't worry too much about it.
It doesn't look like we're checking for this in too many other places so not a problem.